controllers/scopetemplate_controller_test.go

```diff
@@ -235,8 +235,12 @@ var _ = Describe("ScopeTemplate", func() {
 
 						verifyRoleBindings(existingRB, scopeInstance, scopeTemplate)
 
-						roleBindingList = listRoleBinding(namespace.GetName(), 0, labels)
+						labels = map[string]string{scopeInstanceUIDKey: string(scopeInstance.GetUID()),
+							scopeTemplateUIDKey:           string(scopeTemplate.GetUID()),
+							clusterRoleBindingGenerateKey: "test"}
```

Comment:
it looks like we already set the labels previously. I don't think it is necessary to set it again unless the `listRoleBinding()` function is modifying the labels.